ooni/nettest.py

```diff
@@ -567,12 +566,18 @@ def makeMeasurement(self, test_instance, test_method, test_input=None):
         return measurement
 
     @defer.inlineCallbacks
-    def initializeInputProcessor(self):
-        for test_class, test_method in self.testCases:
+    def initialize(self):
+        for test_class, _ in self.testCases:
+            # Initialize Input Processor
             test_class.inputs = yield defer.maybeDeferred(
                 test_class().getInputProcessor
             )
 
+            # Run the setupClass method
+            yield defer.maybeDeferred(
+                test_class.setUpClass
```

Comment:
Because `getInputProcessor` is an instance method, while `setUpClass` is a class method. The former operates on an instance of `test_class`, while the later operates on the `test_class` directly.